### PR TITLE
Fix PHP 7.3 compat in LocalMessageProvider

### PR DESCRIPTION
**Pull request by JeroenDeDauw**
Fixes https://github.com/SemanticMediaWiki/SemanticMediaWiki/issues/5165

### CODE REVIEW

File: SemanticMediaWiki.php [hunk 1]
```diff
@@ -39,6 +39,8 @@ public static function initExtension( $credits = [] ) {
 			define( 'SMW_EXTENSION_LOADED', true );
 		}
 
+		include_once __DIR__ . '/src/Aliases.php';
```
Comment (hexmode) [inline SemanticMediaWiki.php:42]: What does this have to do with 7.3 compat?

**Review by hexmode — COMMENTED** [review]
I thought we said 4.x was going to require 7.4?

File: SemanticMediaWiki.php (hunk 1 above)
Comment (JeroenDeDauw) [reply SemanticMediaWiki.php:42]: Nothing. Thanks. This is what happens when you have two different clones of SMW open and modifying them at the same time.
